src/Sarif.Viewer.VisualStudio/OpenLogFileCommands.cs

```diff
@@ -168,117 +163,80 @@ private async System.Threading.Tasks.Task MenuItemCallbackAsync(object sender, E
 
             if (logFile == null)
             {
-                string title = "Open Static Analysis Results Interchange Format (SARIF) file";
-                string filter = "SARIF files (*.sarif)|*.sarif";
+                FieldInfo[] toolFormatFieldInfos = typeof(ToolFormat).GetFields();
```

Comment:
>GetFields [](start = 70, length = 9)

This will also give you `ToolFormat.None`, and you don't want that one.

Reply:
Ok, I see you deal with that below.

---
In reply to: [505718754](https://github.com/microsoft/sarif-visualstudio-extension/pull/224#discussion_r505718754) [](ancestors = 505718754)